.golangci.yml

```diff
@@ -0,0 +1,23 @@
+linters:
+  enable:
+    - structcheck
+    - varcheck
+    - staticcheck
+    - unconvert
+    - gofmt
+    - goimports
+    - golint
+    - ineffassign
+    - vet
+    - unused
+    - misspell
+  disable:
+    - errcheck
+
+# FIXME: re-enable after fixing GoDoc in this repository
+#issues:
+#  include:
+#    - EXC0002
```

Comment:
Temporarily disabling this check, because there's quite some exported types/functions left to document (will open a tracking issue for that);

```
Error: exported type `CNI` should have comment or be unexported (golint)
Error: exported type `ConfigResult` should have comment or be unexported (golint)
Error: exported type `ConfNetwork` should have comment or be unexported (golint)
Error: exported var `ErrCNINotInitialized` should have comment or be unexported (golint)
Error: exported type `Network` should have comment or be unexported (golint)
Error: exported method `Network.Attach` should have comment or be unexported (golint)
Error: exported method `Network.Remove` should have comment or be unexported (golint)
....
```